src/cheerioManipulation.js

```diff
@@ -87,23 +103,35 @@ const removeSerializerKeys = function ($, vueWrapper) {
  * <input value="Hello World">
  *
  * @param {object} $           The markup as a cheerio object
- * @param {object} vueWrapper  The Vue-Test Utils mounted component wrapper
+ * @param {object} vueWrapper  The Vue-Test Utils mounted component wrapper or Testing Library Vue rendered wrapper
  */
 const addInputValues = function ($, vueWrapper) {
   if (
     globalThis.vueSnapshots?.addInputValues &&
     attributesCanBeStringified(vueWrapper)
   ) {
     debugLogger({ function: 'cheerioManipulation.js:addInputValues' });
-    $('input, textarea, select').each(function (index, element) {
-      const currentKey = $(element).attr(KEY_NAME);
-      const vnode = vueWrapper.find('[' + KEY_NAME + '="' + currentKey + '"]');
-      const value = vnode.element.value;
-      element.attribs.value = swapQuotes(stringify(value));
-      if (['checkbox', 'radio'].includes(element.attribs.type)) {
-        element.attribs.checked = String(vnode.element.checked);
-      }
-    });
+    if (isVueTestUtilsWrapper(vueWrapper)) {
+      $('input, textarea, select').each(function (index, element) {
+        const currentKey = $(element).attr(KEY_NAME);
+        const vnode = vueWrapper.find('[' + KEY_NAME + '="' + currentKey + '"]');
+        const value = vnode.element.value;
+        element.attribs.value = swapQuotes(stringify(value));
+        if (['checkbox', 'radio'].includes(element.attribs.type)) {
+          element.attribs.checked = String(vnode.element.checked);
+        }
+      });
+    } else {
+      const inputs = Array.from(vueWrapper.container.querySelectorAll('input, textarea, select'));
+      inputs.forEach(function (element) {
+        const currentKey = element.getAttribute(KEY_NAME);
+        const vnode = vueWrapper.container.querySelector('[' + KEY_NAME + '="' + currentKey + '"]');
+        element.setAttribute('value', swapQuotes(stringify(vnode.value)));
```

Comment:
This is updating the vueWrapper's element to add this attribute. We DO NOT want to mutate the vueWrapper (causes issues for the user, other than adding the magic key that we later remove).

```suggestion
        $('[' + KEY_NAME + '=' + currentKey ']').attribs.value = swapQuotes(stringify(vnode.value));
```

I *think* that code is right, but double-check it

Instead, we need to look for the ``